### ISSUE TITLE
Python installer fails because there is no requirements.txt

### ISSUE THREAD

**Issue by SineSwiper** [derrian-distro/LoRA_Easy_Training_Scripts#96]
You have a line here to install requirements in the Python installer:

```
    print("installing other requirements")
    subprocess.check_call(f"{python} install -r requirements.txt".split(" "))
```

However, there is no `requirements.txt` file to use, so the process fails:

```
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
```

There's a `requirements_ui.txt`, but that's used by a different part of the process.

**Comment by SineSwiper** [comment]
Actually, I realized this was trying to install requirements for `sd_scripts`, but since [I downloaded a release](https://github.com/derrian-distro/LoRA_Easy_Training_Scripts/releases/tag/UI-Update-v1), it doesn't have the git hooks for submodule updates.

So, the real problem is the existence of those releases.  It should really just redirect to the main GitHub page, because there's no other way to install them besides a `git clone` of the repo.
